app/employeehandler.py

```diff
@@ -96,21 +96,21 @@ async def get(self, request):
     job_role = employee_info.get('jobRole', "-")
 
     job_role['contractStartDate'] = format_to_uk_dates(
-        job_role['contractStartDate'])
+        job_role.get('contractStartDate'))
     job_role['operationalEndDate'] = format_to_uk_dates(
-        job_role['operationalEndDate'])
+        job_role.get('operationalEndDate'))
 
     #  if a job role is returned then format all the dates to uk
     if job_role:
-      if job_role['contractStartDate']:
+      if job_role.get('contractStartDate'):
```

Comment:
There's some duplication here, and there's probably a way to simplify it.

```python
date_fields = ['contractStartDate', 'contractEndDate', 'operationalEndDate']
for field in date_fields:
  if job_role.get(field):
    job_role[field] = format_to_uk_dates(job_role.get(field))
```

Reply:
Have added this more simplified, better code